lib/hexpm_web/controllers/user_controller.ex

```diff
@@ -27,6 +32,11 @@ defmodule HexpmWeb.UserController do
       Packages.accessible_user_owned_packages(user, conn.assigns.current_user)
       |> Packages.attach_versions()
 
+    downloads = Packages.packages_downloads_with_all_views(packages)
+
+    total_downloads =
+      Enum.reduce(downloads, 0, fn {_id, d}, acc -> acc + Map.get(d, "all", 0) end)
```

Comment:
Should we add a specific query for this?

Reply:
I think the list of releases/versions is usually pretty small (<20) so running a reduce over it probably doesnt take much time or effort.